File: QolmsYappli/QolmsYappli/Models/JsonParameter/NoteExaminationHelthAgeJsonParamater.vb
Imports System.Runtime.Serialization

<DataContract()>
<Serializable()>
Public NotInheritable Class NoteExaminationHelthAgeJsonParamater
    Inherits QyJsonParameterBase

#Region "Public Property"

    <DataMember()>
    Public Property Accountkey As Guid = Guid.Empty

    <DataMember()>
    Public Property LoginAt As Date = Date.MinValue

    <DataMember()>
    Public Property healthAgeCalcN As Dictionary(Of Date, Dictionary(Of String, String))


#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="NoteExaminationHelthAgeJsonParamater" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()

        MyBase.New()

    End Sub

#End Region

End Class

File: QolmsJotoWebView/QolmsJotoWebView/Models/JsonParameter/NoteExaminationHelthAgeJsonParamater.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace MGF.QOLMS.QolmsJotoWebView
{
    [DataContract]
    [Serializable]
    public sealed class NoteExaminationHelthAgeJsonParamater : QjJsonParameterBase
    {
        #region "Public Property"

        [DataMember]
        public Guid Accountkey { get; set; } = Guid.Empty;

        [DataMember]
        public DateTime LoginAt { get; set; } = DateTime.MinValue;

        [DataMember]
        public Dictionary<DateTime, Dictionary<string, string>> healthAgeCalcN { get; set; }

        #endregion

        #region "Constructor"

        /// <summary>
        /// <see cref="NoteExaminationHelthAgeJsonParamater" /> クラスの新しいインスタンスを初期化します。
        /// </summary>
        public NoteExaminationHelthAgeJsonParamater()
            : base()
        {
        }

        #endregion
    }

}